Documentation/tutorial/docfx_getting_started.md

```diff
@@ -71,21 +71,35 @@ Now you can view the generated website on http://localhost:8080.
 
 ## 4. Use *DocFX* with a Build Server
 
-*DocFX* can be used in a Continuous Integration environment.
+## 4.1 Steps
 
-Most build systems do not checkout the branch that is being built, but use a `detached head` for the specific commit.  DocFX needs the branch name to implement the `View Source` link in the API documentation.
+*DocFX* can be used in a Continuous Integration (CI) environment. 
+
+Consider a typical scenario: I want to set up a job on a CI service (e.g. [Azure Pipelines](https://azure.microsoft.com/en-us/services/devops/pipelines/)) to build documents automatically and publish to a website (e.g. [GitHub Pages](https://pages.github.com/)). We can achieve this by a script on CI jobs with following steps:
+
+1. clone repo
+2. `choco install docfx`
+3. `docfx {docsetPath}`
+4. push all the content in `docsetPath/_site` to `gh-pages` branch
```

Comment:
Even with the gh-pages branch, don't you have to have the content at the top of the repository?

Might be easier to put instructions on how to update site path in docfx.json

Reply:
Yes, updated. Users can also copy/move file adding some scripts, depending on the scenario.